test/extractors/name_party_extractor_test.rb

```diff
@@ -371,6 +371,17 @@ class NamePartyExtractorTest < ActiveSupport::TestCase
     assert_equal 'ABC', pair[:parties].first
   end
 
+  test 'npc: people seperated by comma and und, one party but surname only 3 letters' do
+    pair = NamePartyExtractor.new("Vorname Nac, Anderer Name und Andre Name ABC", :npc).extract
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.